frontend/app/admin/src/views/organizations/composables/useChargeOrganizationsSheet.ts

```diff
@@ -0,0 +1,22 @@
+import { ComponentWithProperties, usePresent } from '@simonbackx/vue-app-navigation';
+import { StamhoofdFilter } from '@stamhoofd/structures';
+import ChargeOrganizationsView from '../ChargeOrganizationsView.vue';
+
+export function useChargeOrganizationsSheet() {
```

Comment:
Aangezien deze hook maar één keer gebruikt wordt, zou ik deze verplaatsen, bovendien zie ik weinig meerwaarde hierin eerlijk gezegd omdat de hook wrs langere code is dan gewoon rechtstreeks de view te tonen, wat op zich al vrij kort is. Ook de naamgeving die 'present' eigenlijk 'show' heet is niet echt een mooi patroon en maakt het verschil tussen show en present onduidelijker